scripts/build.sh

```diff
@@ -24,6 +24,14 @@ CI_BUILD=false
 VERBOSE=false
 PROJECT_NAME_PATTERNS=
 
+#
+# Source build repo api
+# See https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/RepoApi.md
+#
+DOTNETBUILDFROMSOURCE=false
+DOTNETCORESDKDIR=
+DOTNETBUILDTOOLSDIR=
```

Comment:
`DOTNETBUILDTOOLSDIR` Usage?

Reply:
It is required by the RepoApi, we are not using it.